src/Symfony/Component/JsonPath/Functions/JsonPathFunctionInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Functions;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+interface JsonPathFunctionInterface
+{
+    /**
+     * @param array<string> $args    Array of argument strings as they appear in the expression
```

Comment:
I would even be `list<string>` to be more precise, as they are always provided as a list.